### ISSUE TITLE
Support aux dates

### ISSUE THREAD

**Issue by tbm** [beancount/beancount2ledger#14]
Support aux dates from (configurable) meta-date:

```
2018-03-17 * "Test with transaction aux date"
  aux-date: 2018-03-16
  Assets:Test                        10.00 EUR
  Equity:Opening-Balance            -10.00 EUR
```

**Comment by tbm** [comment]
```
2018-03-18 * "Test with posting aux date"
  Assets:Test                        10.00 EUR
    aux-date: 2018-03-13
  Equity:Opening-Balance

2018-03-18 * "Test with transaction and posting aux date"
  aux-date: 2018-03-17
  Assets:Test                        10.00 EUR
    aux-date: 2018-03-13
  Equity:Opening-Balance
```

**Comment by tbm** [comment]
and posting dates:

```
2019-01-16 * "Posting date using comment"
  Assets:Test                        10.00 EUR
    date: 2017-03-04
  Equity:Opening-Balance            -10.00 EUR
    date: 2017-07-08
```

**Comment by tbm** [comment]
Fixed in commit da32a4c443b50e3225c1a34290e1a8bb4cfc0e24
